symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/message/MessageService.java

```diff
@@ -386,4 +406,22 @@ private static Long getEpochMillis(Instant instant) {
   private <T> T executeAndRetry(String name, SupplierWithApiException<T> supplier) {
     return RetryWithRecovery.executeAndRetry(retryBuilder, name, supplier);
   }
+  
+  private File createTemporaryAttachmentFile(Attachment attachment) {
+    if (attachment == null) {
+      return null;
+    }
+    try {
+      String[] fileNameArrays = attachment.filename().split("\\.");
+      if (fileNameArrays.length < 2) {
```

Comment:
isn't it something we could check when building the attachment as to create only valid attachment?